.travis.yml

```diff
@@ -9,5 +9,4 @@ services:
   - redis-server
 rvm:
   - 2.2.4
-  - 2.1.8
```

Comment:
This sucks, but Travis just won't install the json gem, so I don't know what else to do.  I'm not downloading some Docker image